### PR TITLE
Update slides

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: slides/geocomp-transport.Rmd [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```{r, echo=FALSE}
+# download.file("https://raw.githubusercontent.com/Robinlovelace/geocompr/master/refs.bib", "refs.bib")
+# download.file("https://raw.githubusercontent.com/Robinlovelace/geocompr/master/packages.bib", "packages.bib")
+```
+
+```{r, echo=FALSE}
+library(methods)
+```
+
+Although it has academic origins, R is now widely used for real-world applications in areas as diverse as finance, emidemiology and conservation.
```
Comment (jannes-m) [inline slides/geocomp-transport.Rmd]: epidemiology

Comment (jannes-m) [reply slides/geocomp-transport.Rmd]: don't leave out location analysis -:)

File: slides/geocomp-transport.Rmd [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+This talk will make the case that transport research, and transport planning in particular, are domains in which uptake of R would be particularly beneficial.
+Transport planning uses spatial and temporal datasets and relies on transparency, debate of options and visualisation to ensure evidence-based interventions are commissioned.
+R excels in each of these areas and benefits from recent packages such as **sf**.
+To illustrate the point the talk will describe work commissioned by the UK's Department for Transport to develop a Propensity to Cycle Tool (PCT), which is now being used around England to help design strategic cycle networks and improve the effectiveness of precious public investment in transport infrastructure.
+Based on the experience developing the PCT we will discuss the limitations of R as a tool for transport planning, and the potential of recent developments in packages such as **stplanr** and **dodgr** packages to address them.
+The talk will conclude wby outlining ways in which R, and open source software in general, can be used
```
Comment (jannes-m) [inline slides/geocomp-transport.Rmd]: typo: wby should be by

Comment (jannes-m) [reply slides/geocomp-transport.Rmd]: Nice abstract, really applied, I like it, not just academic stuff!

File: slides/geocomp-transport.Rmd [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# download.file("https://raw.githubusercontent.com/Robinlovelace/geocompr/master/refs.bib", "refs.bib")
+# download.file("https://raw.githubusercontent.com/Robinlovelace/geocompr/master/packages.bib", "packages.bib")
+```
+
+```{r, echo=FALSE}
+library(methods)
+```
+
+Although it has academic origins, R is now widely used for real-world applications in areas as diverse as finance, emidemiology and conservation.
+In other areas R has been slow to catch-on, despite offering advantages of speed, reproducibility and flexibility over some proprietary products.
```
Comment (jannes-m) [inline slides/geocomp-transport.Rmd]: which areas are you referring to?

Comment (jannes-m) [reply slides/geocomp-transport.Rmd]: well, speed...

File: slides/gis_with_r_start.Rmd [hunk 4]
```diff
@@ -549,7 +552,7 @@ dev.off()
 ### Extract
 
 ```{r}
-ham_cities_sp = as(ham_cities_sf, "Spatial")
+ham_cities_sp = as(ham_cities_sf, "Spatial")
```
Comment (Robinlovelace) [inline slides/gis_with_r_start.Rmd]: Nudge @Nowosad any update on this? Couldn't see where that dataset came from..

**Pull request merged**